bundlewrap/cmdline/parser.py

```diff
@@ -1227,4 +1285,16 @@ def build_parser_bw():
     # bw zen
     parser_zen = subparsers.add_parser("zen")
     parser_zen.set_defaults(func=bw_zen)
+
+    if shell_completion:
+        # bw generate_completions
+        help_generate_completions = _("Generates the shell completion file")
+        parser_apply = subparsers.add_parser(
```

Comment:
This definitely should not be `parser_apply`. Can you rename that? Otherwise this looks ready to merge.